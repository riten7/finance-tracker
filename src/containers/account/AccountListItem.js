import React from 'react';
import './account.css';

const AccountListItem = (props) => {
  const { name, balance } = props.account;
  return (
    <div className="account-item">
      <span>{name}</span>
      <span className={`account-item__balance ${balance > 0 ? 'green': 'red'}`}>{`NPR. ${balance}`}</span>
    </div>
  );
}

export default AccountListItem;