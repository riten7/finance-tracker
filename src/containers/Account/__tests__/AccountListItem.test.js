import React from 'react';
import { mount } from "enzyme";
import AccountListItem from '../AccountListItem';

describe("AccountListItem component", () => {
  let wrapper;
   it("Component renders properly", () => {
    wrapper = mount(<AccountListItem account={{
        name: "dtrdtr",
        type: "cash",
        balance: 131406817,
        id: "6zb98plvkbr"
      }}/>);
    expect(wrapper.html()).toMatchSnapshot();
  });
});