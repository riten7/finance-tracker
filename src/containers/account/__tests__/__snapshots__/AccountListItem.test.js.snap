// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountListItem component Component renders properly 1`] = `"<div class=\\"account-item\\"><span>dtrdtr</span><span class=\\"account-item__balance green\\">NPR. 131406817</span></div>"`;
